vault.md

```diff
@@ -95,4 +95,107 @@ vault kv put kv/hello target=world
 ```bash
 helm uninstall vault -n vault
 kubectl delete pvc -n vault -lapp.kubernetes.io/name=vault
+```
+
+or 
+
+```bash
+./vault.sh remove
+```
+
+## Using Dev Services for Vault in Primaza
+
+When testing or running in dev mode Quarkus can provide you with a zero-config Vault out of the box. 
+Since the Vault extension is included in the classpath, Quarkus will provide the Vault server inside a Docker container. 
+So, to use the Vault provided by Quarkus, start the application in dev mode: 
+
+```bash
+mvn quarkus:dev
+```
+
+Tape `h` key as indicated in the prompt for more options:  
+
+```bash
+The following commands are currently available:
+
+== Continuous Testing
+
+[r] - Resume testing
+[o] - Toggle test output (disabled)
+
+== Dev Services
+
+[c] - Show dev services containers
+[g] - Follow dev services logs to the console (disabled)
+
+== Exceptions
+
+[x] - Opens last exception in IDE (None)
+
+== HTTP
+
+[w] - Open the application in a browser
+[d] - Open the Dev UI in a browser
+
+== System
+
+[s] - Force restart
+[i] - Toggle instrumentation based reload (disabled)
+--
+
+
+```
+Then tape `c` in order to show the Dev Services containers running. You should see something like:
+
+```bash
+
+== Dev Services
+
+  Injected Config:  quarkus.vault.url=http://localhost:64966, quarkus.vault.authentication.client-token=root
+
+```
+
+If you want to use this Vault in command line for creating or reading secrets, you can configure the address and token as follows:
```

Comment:
`If you want to use this Vault in command line` -> `If you want to it using the vault client within a terminal, export the following ENV variables:`